Makefile

```diff
@@ -90,8 +90,8 @@ help: ## Display this help
 config: SHELL:=$(RUN_IN_DEV_SHELL)
 config: images/dev-env/.dockerbuilt ## Generate CRD manifests with controller-gen
 	cd ./api ;\
-		controller-gen crd:trivialVersions=true,preserveUnknownFields=false,crdVersions=v1 rbac:roleName=manager-role paths="./..." output:crd:artifacts:config=../config/crd/bases ;\
-		controller-gen crd:trivialVersions=true,preserveUnknownFields=false,crdVersions=v1 rbac:roleName=manager-role paths="./..." output:crd:artifacts:config=../helm/chaos-mesh/crds ;
```

Comment:
Just a note: `crd:trivialVersions=true,preserveUnknownFields=false` is not working when the default CRDs are `v1`.

Reply:
I checked the source code of `controller-gen` and `trivialVersions` and `preserveUnknownFields` were removed as  options in this PR https://github.com/kubernetes-sigs/controller-tools/pull/607. The first release to include the change is https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.7.0 released ~ 2 years ago.

Reply:
Yes, I learned about it through this PR as well. But I didn't find anything about it on the kubebuilder documentation, so I'm leaving a note here.